src/redux/news/news.reducer.js

```diff
@@ -2,14 +2,20 @@ import {
   SET_NEWS,
   SET_NEWS_LOADING,
   SET_ARTICLE,
-  SET_NEWS_ERROR
+  SET_NEWS_ERROR,
+  SET_CURRENT_PAGE,
+  SET_NEWS_PER_PAGE,
+  SET_PAGES_COUNT
 } from './news.types';
 
 const initialState = {
   list: [],
   newsArticle: null,
   newsLoading: false,
-  newsError: null
+  newsError: null,
+  currentPage: 0,
```

Comment:
Please, put all of the pagination fields inside its own `pagination` object

Reply:
Changed!